kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -493,10 +493,13 @@ define([
 
                 let filePathRows = ui.getElements(`${cssClassType}-fields-row`);
 
-                filePathRows.forEach((filePathRow) => {
```

Comment:
this was returning weirdly because it's a synchronous call wrapping a bunch of promises, so i changed it to be a promise.all call. also needed to have updateRows called AFTER it resolved so updated that as well